saleor/order/utils.py

```diff
@@ -290,42 +303,53 @@ def add_gift_card_to_order(order, gift_card, total_price_left):
     return total_price_left
 
 
-def change_order_line_quantity(user, line, old_quantity, new_quantity):
+def change_order_line_quantity(context, line, old_quantity, new_quantity):
     """Change the quantity of ordered items in a order line."""
     if new_quantity:
         line.quantity = new_quantity
-        total_price_net_amount = line.quantity * line.unit_price_net_amount
-        total_price_gross_amount = line.quantity * line.unit_price_gross_amount
+        line.save(update_fields=["quantity"])
+        net = line.unit_price.net.amount
```

Comment:
Why? what was wrong with this part of code? This logic was working well with the draft order so I don't understand why this changes are required.
Also unit_price returned from plugins will never be a None.